ftw/referencewidget/resources/refwidget.js

```diff
@@ -0,0 +1,167 @@
+$(function() {
+    $(document).on('click', '.referencewidget button', openOverlay);
+
+    $(document).on('click', '.refbrowser .path span', jump_to);
+    $(document).on('click', '.refbrowser .search button', search);
+    $(document).on('keypress', '.refbrowser .search input', function(event){
+        if(event.which == 13) {search(event);}});
+
+    $(document).on('change', '.refbrowser .listing input.ref-checkbox', checkbox_flipped);
+    $(document).on('click', '.refbrowser button.cancel', function(){$('.refbrowser').remove()});
+    $(document).on('click', '.refbrowser .ref_list_entry', switch_level);
+    $(document).on('click', '.refbrowser .listing input.ref-checkbox', function(e) {e.stopPropagation();});
+    request_data = {}
+    url = location.protocol + '//' + location.host + location.pathname;
+    widget_url = ""
```

Comment:
Please use jslint or jshint or something that tells you that you need to make a semicolon at the end of this line.